crates/node/p2p/src/gossip/driver.rs

```diff
@@ -224,7 +224,7 @@ where
 
     fn handle_gossip_event(&mut self, event: Event) -> Option<OpNetworkPayloadEnvelope> {
         match event {
-            Event::Gossipsub(e) => return self.handle_gossipsub_event(e),
+            Event::Gossipsub(e) => return self.handle_gossipsub_event(*e),
```

Comment:
We could just pass in a reference here and change the parameter type, but for the sake of keeping this bump pr lean, this is fine

Reply:
Ah I see it's boxed